config/prod/s3-to-glue-lambda.yaml

```diff
@@ -0,0 +1,10 @@
+template:
+  type: sam
+  path: src/lambda_function/template.yaml
+  artifact_bucket_name: !stack_output_external recover-cloudformation-bucket
+  artifact_prefix: 'recover/{{ stack_group_config.namespace }}/lambda'
+stack_name: '{{ stack_group_config.namespace }}-lambda-S3ToGlue'
+stack_tags: {{ stack_group_config.default_stack_tags }}
+parameters:
+  Namespace: {{ stack_group_config.namespace }}
+  S3SourceBucketName: recover-pilot-data
```

Comment:
We shouldn't have access to the pilot data in prod. Use `{{ stack_group_config.source_bucket }}` instead. See config/config.yaml, `source_bucket` is the shared bucket.

Reply:
The creation of this stack in prod is dependent on [ETL-318](https://sagebionetworks.jira.com/browse/ETL-318) though?

Reply:
The whole prod pipeline doing anything useful is dependent on ETL-318. It's more of a placeholder than anything else, although a slightly more correct placeholder than recover-pilot-data.

Reply:
More importantly -- we don't want anything in prod to have access to anything that's not FISMA-moderate compliant. Our dev account is not compliant.

Reply:
Make sense, got it!